styles1.css

```diff
@@ -279,6 +279,66 @@ p {
 }
 
 /* --------------HERE STARTS MY RECENT WORKS------------------- */
+.popup {
+  display: none;
+  background-color: #fff;
+  flex-direction: column;
+  position: fixed;
+  top: 0;
+  margin: 5% 10%;
+  padding: 2%;
+}
```

Comment:
There are two issues regarding the popup that needs attention 🧐

As you can see, the popup is not centered horizontally. Kindly make sure to address this to make your project as close as to Figma's 🔥 Please also take a look at the margins for your mobile popup 🤓 You could give the text a little bit more air with some margins 😉

### Your design 
![Captura](https://user-images.githubusercontent.com/15807343/151094662-fe6b65a4-642e-40d5-9872-f7c4b9bf78d4.PNG)

### Figma's design 
![Captura2](https://user-images.githubusercontent.com/15807343/151094797-fa4af27c-17ec-428d-b77a-3c43003ae41b.PNG)

#### The second issue is about the design when the popup is opened. It's super important to follow the design and make our background fuzzy to show clearly the difference in the UI. 

### Your design 
![Captura5](https://user-images.githubusercontent.com/15807343/151095405-a2d11077-c59c-46da-86ea-7c4f356077fb.PNG)

The projects section and the popup overlaps(rest of the page as well) in this image and doesn't help the user experience 😥 Please address this issue so you can go on to the next project 🙌

### Figma's design 
![image](https://user-images.githubusercontent.com/15807343/151095210-9ab63996-cd8e-4a11-85e9-8585d312500e.png)